addons/mail/static/src/models/thread/thread.js

```diff
@@ -542,6 +618,38 @@ function factory(dependencies) {
             }
         }
 
+        /*
+         * Returns threads that match the given search term.
```

Comment:
Would clarify the context this method is expected to be called. It only works with channels, and private channel only suggest itself for reasons in how this method is expected to be used.

Reply:
Should we make this limitation though? I feel like it would be extremely useful (ctrl+k) to be able to find chatter too based on their name.
But I guess it's fair to comment this in stable while it is indeed limited to channels.